### ISSUE TITLE
Runtime launch configuration

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-distro#869]
I moved to the last master branch with all the stuff regarding ESH => OH.
When I start openHAB inside Eclipse, it looks like the launch configuration is no more ready for a startup. Lots of requirements are missing.
Can someone provide a working launch configuration please ? It would help a lot all non expert of all available bundles.

**Comment by lolodomo** [comment]
After selecting some bundles, here is the last errors:

![image](https://user-images.githubusercontent.com/10186704/52600680-b2043580-2e5c-11e9-85e7-9993bfbe75c0.png)

Where are these bundles now ?

**Comment by kaikreuzer** [comment]
The IDE setup should not give you any option to do "core" development anymore - using projects from openhab-core in your workspace simply won't work.

The IDE setup is meant to do add-on development and this should work as expected.

Wrt to openhab-core development, see https://github.com/openhab/openhab-core/issues/513; there isn't yet any official documentation though.

**Comment by lolodomo** [comment]
Currently my purpose is only testing a new binding. I updated all the repos. I started the run configuration, selected my binding and then openHAB is not starting.
The question is: how can I test a binding inside Eclipse now ? I can't expect running OH inside Eclipse anymore ?

**Comment by kaikreuzer** [comment]
Ok, so you are saying that you do not have any org.openhab.core bundles in your workspace?
Did you make sure to reload/update the target platform? With https://github.com/openhab/openhab-distro/pull/861, everything should be in place to launch openHAB from within the IDE, just as it has always been.

**Comment by lolodomo** [comment]
No core bundles selected in my workspace, only my binding.
I will reload the target platform this evening (did not do it yesterday) and report.

**Comment by lolodomo** [comment]
Ok I have edit/reload the target platform.
Now in plug-ins I just select my additional binding in my workspace and then select the bundles in target platform that are required by my binding. I finally get this window:
![image](https://user-images.githubusercontent.com/10186704/52665735-948ea480-2f0c-11e9-9fcd-65e510c6e8b2.png)
Then I push continue and here are the logs:
```
!SESSION 2019-02-12 21:24:58.688 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_131
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=fr_FR
Framework arguments:  -application org.eclipse.ui.ide.workbench
Command-line arguments:  -application org.eclipse.ui.ide.workbench -data D:\dev\openhab2\ws/../runtime-org.openhab.runtime.product.product -dev file:D:/dev/openhab2/ws/.metadata/.plugins/org.eclipse.pde.core/openHAB_Runtime/dev.properties -os win32 -ws win32 -arch x86_64 -consoleLog -console

!ENTRY org.eclipse.jetty.osgi.boot 4 0 2019-02-12 21:24:59.532
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.osgi.boot [57]
  Unresolved requirement: Import-Package: org.eclipse.jetty.annotations; version="[9.4.11,10.0.0)"; resolution:="optional"
  Unresolved requirement: Import-Package: org.eclipse.jetty.deploy; version="[9.4.11,10.0.0)"
    -> Export-Package: org.eclipse.jetty.deploy; bundle-symbolic-name="org.eclipse.jetty.deploy"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.deploy.graph,org.eclipse.jetty.server,org.eclipse.jetty.server.handler,org.eclipse.jetty.util,org.eclipse.jetty.util.annotation,org.eclipse.jetty.util.component,org.eclipse.jetty.util.resource,org.eclipse.jetty.xml"
       org.eclipse.jetty.deploy [53]
         Unresolved requirement: Import-Package: org.eclipse.jetty.jmx; version="[9.4.11,10.0.0)"; resolution:="optional"
         Unresolved requirement: Import-Package: org.eclipse.jetty.server; version="[9.4.11,10.0.0)"
  Unresolved requirement: Import-Package: org.eclipse.jetty.deploy.bindings; version="[9.4.11,10.0.0)"
    -> Export-Package: org.eclipse.jetty.deploy.bindings; bundle-symbolic-name="org.eclipse.jetty.deploy"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.deploy,org.eclipse.jetty.deploy.graph,org.eclipse.jetty.server"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.

!ENTRY org.eclipse.jetty.client 4 0 2019-02-12 21:24:59.678
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.client [52]
  Unresolved requirement: Import-Package: org.eclipse.jetty.http; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.deploy 4 0 2019-02-12 21:24:59.693
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.deploy [53]
  Unresolved requirement: Import-Package: org.eclipse.jetty.jmx; version="[9.4.11,10.0.0)"; resolution:="optional"
  Unresolved requirement: Import-Package: org.eclipse.jetty.server; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.osgi.httpservice 4 0 2019-02-12 21:24:59.693
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.osgi.httpservice [58]
  Unresolved requirement: Import-Package: org.eclipse.jetty.server; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.proxy 4 0 2019-02-12 21:24:59.709
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.proxy [59]
  Unresolved requirement: Import-Package: org.eclipse.jetty.client; version="[9.4.11,10.0.0)"
    -> Export-Package: org.eclipse.jetty.client; bundle-symbolic-name="org.eclipse.jetty.client"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.client.api,org.eclipse.jetty.client.util,org.eclipse.jetty.http,org.eclipse.jetty.io,org.eclipse.jetty.util,org.eclipse.jetty.util.annotation,org.eclipse.jetty.util.component,org.eclipse.jetty.util.log,org.eclipse.jetty.util.ssl,org.eclipse.jetty.util.thread"
       org.eclipse.jetty.client [52]
         Unresolved requirement: Import-Package: org.eclipse.jetty.http; version="[9.4.11,10.0.0)"
  Unresolved requirement: Import-Package: org.eclipse.jetty.client.api; version="[9.4.11,10.0.0)"
    -> Export-Package: org.eclipse.jetty.client.api; bundle-symbolic-name="org.eclipse.jetty.client"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.http,org.eclipse.jetty.util"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.rewrite 4 0 2019-02-12 21:24:59.709
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.rewrite [60]
  Unresolved requirement: Import-Package: org.eclipse.jetty.http; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.webapp 4 0 2019-02-12 21:24:59.709
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.webapp [65]
  Unresolved requirement: Import-Package: org.eclipse.jetty.http; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.websocket.client 4 0 2019-02-12 21:24:59.709
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.websocket.client [67]
  Unresolved requirement: Import-Package: org.eclipse.jetty.client.api; version="[9.4.11,10.0.0)"
    -> Export-Package: org.eclipse.jetty.client.api; bundle-symbolic-name="org.eclipse.jetty.client"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.http,org.eclipse.jetty.util"
       org.eclipse.jetty.client [52]
         Unresolved requirement: Import-Package: org.eclipse.jetty.http; version="[9.4.11,10.0.0)"
  Unresolved requirement: Import-Package: org.eclipse.jetty.client; version="[9.4.11,10.0.0)"
    -> Export-Package: org.eclipse.jetty.client; bundle-symbolic-name="org.eclipse.jetty.client"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.client.api,org.eclipse.jetty.client.util,org.eclipse.jetty.http,org.eclipse.jetty.io,org.eclipse.jetty.util,org.eclipse.jetty.util.annotation,org.eclipse.jetty.util.component,org.eclipse.jetty.util.log,org.eclipse.jetty.util.ssl,org.eclipse.jetty.util.thread"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.eclipse.jetty.websocket.common 4 0 2019-02-12 21:24:59.725
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.websocket.common [68]
  Unresolved requirement: Import-Package: org.eclipse.jetty.io; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)
osgi> 
!ENTRY org.eclipse.jetty.xml 4 0 2019-02-12 21:24:59.725
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.eclipse.jetty.xml [69]
  Unresolved requirement: Import-Package: org.eclipse.jetty.util; version="[9.4.11,10.0.0)"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)

!ENTRY org.jupnp 4 0 2019-02-12 21:24:59.725
!MESSAGE FrameworkEvent ERROR
!STACK 0
org.osgi.framework.BundleException: Could not resolve module: org.jupnp [80]
  Unresolved requirement: Import-Package: org.eclipse.jetty.client.api; version="[9.3.14,10.0.0)"
    -> Export-Package: org.eclipse.jetty.client.api; bundle-symbolic-name="org.eclipse.jetty.client"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.http,org.eclipse.jetty.util"
       org.eclipse.jetty.client [52]
         Unresolved requirement: Import-Package: org.eclipse.jetty.http; version="[9.4.11,10.0.0)"
  Unresolved requirement: Import-Package: org.eclipse.jetty.client; version="[9.3.14,10.0.0)"
    -> Export-Package: org.eclipse.jetty.client; bundle-symbolic-name="org.eclipse.jetty.client"; bundle-version="9.4.11.v20180605"; version="9.4.11"; uses:="org.eclipse.jetty.client.api,org.eclipse.jetty.client.util,org.eclipse.jetty.http,org.eclipse.jetty.io,org.eclipse.jetty.util,org.eclipse.jetty.util.annotation,org.eclipse.jetty.util.component,org.eclipse.jetty.util.log,org.eclipse.jetty.util.ssl,org.eclipse.jetty.util.thread"

	at org.eclipse.osgi.container.Module.start(Module.java:444)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1628)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1608)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1579)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1522)
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230)
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340)
2019-02-12 21:24:59.943 [DEBUG] [.core.common.ThreadPoolManager:161  ] - Created thread pool 'safeCall' with size 5
2019-02-12 21:25:00.037 [DEBUG] [.e.s.c.i.i18n.I18nProviderImpl:180  ] - No language set, setting locale to 'null'.
2019-02-12 21:25:00.037 [DEBUG] [.e.s.c.i.i18n.I18nProviderImpl:162  ] - Error setting measurement system for value 'null'.
2019-02-12 21:25:00.100 [DEBUG] [.core.common.ThreadPoolManager:131  ] - Created scheduled thread pool 'scheduler' of size 5
2019-02-12 21:25:00.115 [DEBUG] [.core.common.ThreadPoolManager:161  ] - Created thread pool 'ConfigStatusService' with size 5
2019-02-12 21:25:00.115 [DEBUG] [.i.f.FirmwareUpdateServiceImpl:141  ] - Modifying the configuration of the firmware update service.
2019-02-12 21:25:00.115 [DEBUG] [.i.f.FirmwareUpdateServiceImpl:419  ] - Given time unit null is not supported. Will keep current configuration.
2019-02-12 21:25:00.131 [DEBUG] [.core.common.ThreadPoolManager:131  ] - Created scheduled thread pool 'common' of size 5

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.633
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.client_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.deploy_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.osgi.boot_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.osgi.httpservice_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.proxy_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.rewrite_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.webapp_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.websocket.client_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.websocket.common_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.eclipse.jetty.xml_9.4.11.v20180605.jar was not resolved.

!ENTRY org.eclipse.osgi 4 0 2019-02-12 21:25:00.634
!MESSAGE Bundle initial@reference:file:plugins/org.jupnp_2.5.1.jar was not resolved.
```

**Comment by lolodomo** [comment]
If someone could explain the new way to run openHAB in Eclipse to test new or updated bindings, that would be very appreciated.

**Comment by kaikreuzer** [comment]
@lolodomo I have now managed to get the whole runtime stack up and running through the launch config and have created https://github.com/openhab/openhab-distro/pull/872 for those changes.

Unfortunately, I also see the three equinox validation errors when launching, which look like a PDE bug to me - but if one simply ignores them, it all starts as expected.

**Comment by lolodomo** [comment]
@kaikreuzer : thank you Kai, I was able to start openHAB now.

Remains 2 questions:

1. When I try to exit from Eclipse console using the exit command as usual, it now fails with this error (only found option is to use the Terminate button):

```
osgi> exit
Really want to stop Equinox? (y/n; default=y)  y

gogo: SecurityException: Prevented System.exit() call.
```

2. The default launch setup does not include any UI like Basic UI or Paper UI. The UIs are even not present in the target platform and the only option I found was to select them from my workspace (I selected the new ones). Is it expected ?
